data/map systems.txt

```diff
@@ -7273,6 +7775,57 @@ system "Bote Asu"
 			distance 601
 			period 38.6987
 
+system Botein
+	pos -236.66667 106.44445
+	government Syndicate
+	link Ankaa
+	link Electra
+	link Puppis
+	link Ruchbah
+	asteroids "small rock" 18 3.087
+	asteroids "large rock" 1 1.764
+	asteroids "small metal" 1 2.52
+	asteroids "medium metal" 9 2.898
+	asteroids "large metal" 2 2.016
+	minables aluminum 6 2.9659099
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Merchants" 550
	fleet "Large Core Merchants" 650
	fleet "Small Syndicate" 950
	fleet "Large Syndicate" 3000
	fleet "Small Core Pirates" 1900
	fleet "Large Core Pirates" 3200
```
Based on fleet frequencies in Ruchbah and Ankaa, between which this is the shortest route. Pirate fleets slightly more frequent than those systems would indicate, due to some assumed pirate traffic down the back routes.